perun-core/src/test/java/cz/metacentrum/perun/core/entry/ModulesUtilsEntryIntegrationTest.java

```diff
@@ -792,6 +794,31 @@ public void countUserFacilityQuotas() throws Exception {
 		assertEquals(expectedResult, result);
 	}
 
+	@Test
+	public void getUserByLoginInNamespace() throws Exception {
+		System.out.println(CLASS_NAME + "getUserByLoginInNamespace");
+
+		// create user
+		User user = new User();
+		user.setFirstName("Firstname");
+		user.setLastName("Lastname");
+		user = perun.getUsersManagerBl().createUser(sess, user);
+		assertNotNull(user);
+
+		User userByLogin = perun.getModulesUtilsBl().getUserByLoginInNamespace(sess, "testlogin", "einfra");
+		assertNull("No user should be found.", userByLogin);
+
+		// create login in einfra namespace
+		String attrName = AttributesManager.NS_USER_ATTR_DEF + ":" + AttributesManager.LOGIN_NAMESPACE + ":einfra";
```

Comment:
I would create some new testing login namespace instead of using 'einfra' which may not exists.

Reply:
We have "dummy" login namespace / module for the purpose of the tests.

Reply:
Done, I create "dummy" login namespace attribute, if it is not already created.